docs/hub/adding-a-model.md

```diff
@@ -116,3 +116,78 @@ Models trained with 🤗 Transformers will generate [TensorBoard traces](https:/
 Beyond 🤗 Transformers, any repository that contains TensorBoard traces (filenames that contain `tfevents`) is categorized with the [`TensorBoard` tag](https://huggingface.co/models?filter=tensorboard). As a convention we suggest that you save traces under the `runs/` subfolder. The "Training metrics" tab then makes it easy to review charts of the logged variables, like the loss or the accuracy.
 
 ![Training metrics tab on a model's page, with TensorBoard](/docs/assets/hub/tensorboard.png)
+
```

Comment:
## Why we implemented Signed commits on the Hugging Face hub

`git` has an authentication layer to control who can push commits to a repo, but it does not authentify the actual commit authors.

In other words, you can commit changes as `Elon Musk <elon@tesla.com>`, push them to your preferred Git host (for instance github.com) and your commit will link to Elon's GitHub profile. (Try it! But don't blame us if Elon gets mad at you for impersonating him)

See this post by Ale Segala for more context: [How (and why) to sign Git commits](https://withblue.ink/2020/05/17/how-and-why-to-sign-git-commits.html)

Using GPG however – and a key server – you can "prove" that a commit was authored by *you*.

We'll explain how to set this up on hf.co below. The reasons we implemented GPG signing was:
- to provide finer-grained security, especially as more and more Enterprise users rely on the Hub (in the future you'll be able to disallow non-signed commits)
- coupled with our model [eval results metadata](https://github.com/huggingface/huggingface_hub/blame/main/modelcard.md), this enables a crypto-secure trustable source of ML benchmarks 🤯 

The Pro Git book is, as usual, a good resource about commit signing: [Pro Git: Signing your work](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work).

Reply:
We implemented more than signed commits, it's verified commits no? (since anyone can create a gpg key with anyone's email)

Reply:
hmm I think it is understandable like this, in simple terms.

I can update to `Why we implemented Signed commits verification on the Hugging Face hub` if we want to be more precise about it? What do other people think?

Reply:
I like `Why we implemented signed commits on the Hugging Face Hub` because it's shorter, and I think verification is implied when you sign a commit. I think we can briefly introduce GPG a bit more (unless this is a commonly known thing), without redirecting the user. Maybe we can have something like:

You can prove a commit was authored by you, using GNU Privacy Guard (GPG) and a key server. GPG is a cryptographic tool used to verify the authenticity of a message's origin.